config/config.go

```diff
@@ -468,6 +468,9 @@ type NetworkConfigData struct {
 
 	// Sessions configures session management.
 	Sessions SessionsConfig `json:"sessions"`
+
+	// Webrtc indicates whether or not to enable WebRTC support.
+	WebRTC bool `json:"webrtc"`
```

Comment:
I think it would be easier to just turn it on by default like we discussed

Reply:
And no config option

Reply:
@edaniels @bhaney Added default value for webrtc flag instead of config option.